src/Lykke.Job.Messages/Sagas/BlockchainOperationsSaga.cs

```diff
@@ -66,6 +66,9 @@ public async Task Handle(InterestPayout.MessagingContract.PayoutCompletedEvent e
             var isTradingWallet = evt.WalletId == evt.ClientId;
             var walletId = isTradingWallet? (Guid?)null : Guid.Parse(evt.WalletId);
             var clientId = Guid.Parse(evt.ClientId);
+
+            if (!evt.ShouldNotifyUser)
+                return;
```

Comment:
Let's log this. With context - wallet id, client id, asset id, operation id

Reply:
`+`